extras/retrofit2/src/test/java/org/asynchttpclient/extras/retrofit/AsyncHttpRetrofitIntegrationTest.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2017 AsyncHttpClient Project. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.asynchttpclient.extras.retrofit;
+
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+import retrofit2.converter.scalars.ScalarsConverterFactory;
+import rx.schedulers.Schedulers;
+
+import java.io.IOException;
+import java.util.Collection;
+
+/**
+ * All tests in this test suite are disabled, because they call functionality of github service that is
+ * rate-limited.
+ */
+@Slf4j
+public class AsyncHttpRetrofitIntegrationTest {
+    private static final String OWNER = "AsyncHttpClient";
+    private static final String REPO = "async-http-client";
+
+    protected static AsyncHttpClient httpClient = createHttpClient();
+
+    private static AsyncHttpClient createHttpClient() {
+        val config = new DefaultAsyncHttpClientConfig.Builder()
+                .setCompressionEnforced(true)
+                .setTcpNoDelay(true)
+                .setKeepAlive(true)
+                .setPooledConnectionIdleTimeout(120_000)
+                .setFollowRedirect(true)
+                .setMaxRedirects(5)
+                .build();
+
+        return new DefaultAsyncHttpClient(config);
+    }
+
+    @AfterSuite
+    void cleanup() throws IOException {
+        httpClient.close();
+    }
+
+    @Test(enabled = false)
```

Comment:
They're using Github API, which is rate-limited and github may not be up all the time. Do you want me to convert this test so that it would start `wiremock` or  similar  webserver on local host and perform http calls against it?